proposals/description/fip_76a.ts

```diff
@@ -0,0 +1,41 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_x: ProposalDescription = {
```

Comment:
Should this `const fip_x` be `const fip_76`? It doesn't matter really because it's a default export, but would be more semantically/logically consistent. 

Also as this is the default export for the file, why don't we just default export it here rather than at the bottom of the file. `export default const fip_76: ProposalDescription = `

Reply:
I'll change but hopefully later we transition these to JSON files. Would allow front-end to parse them, too